HMCL/src/main/resources/assets/lang/I18N_zh.properties

```
@@ -1001,11 +1002,18 @@ repositories.chooser.title=選取 JavaFX 下載源
 
 resourcepack=資源包
 resourcepack.add=新增資源包
-resourcepack.manage=資源包管理
-resourcepack.download=下載資源包
 resourcepack.add.failed=新增資源包失敗
+resourcepack.compatible=適用於此版本
 resourcepack.delete.failed=刪除資源包失敗
+resourcepack.download=下載資源包
 resourcepack.download.title=資源包下載 - %1s
+resourcepack.manage=資源包管理
+resourcepack.warning.invalid=資源包元數據無效
+resourcepack.warning.manipulate=資源包的加載可能受到模組影響，實際效果可能與預期不符。\n可能需要在遊戲內重新確認。
+resourcepack.warning.missing_game_meta=遊戲版本元數據缺失
+resourcepack.warning.missing_pack_meta=資源包元數據缺失
+resourcepack.warning.too_new=爲更新的遊戲版本打造
+resourcepack.warning.too_old=爲更老的遊戲版本打造
```

目前 HMCL 繁体大量采用台湾习惯。下面的建议中包括不限于用词和字形。

```suggestion
resourcepack.manage=資源包管理
resourcepack.warning.invalid=資源包中繼資料無效
resourcepack.warning.manipulate=資源包的載入可能受到模組影響，實際效果可能與預期不符。\n可能需要在遊戲內重新確認。
resourcepack.warning.missing_game_meta=遊戲版本中繼資料缺失
resourcepack.warning.missing_pack_meta=資源包中繼資料缺失
resourcepack.warning.too_new=為更新的遊戲版本製作
resourcepack.warning.too_old=為更老的遊戲版本製作
```